packages/manager/modules/core/src/index.js

```diff
@@ -185,6 +187,9 @@ angular
     $httpProvider.interceptors.push('OvhSsoAuthInterceptor');
     $httpProvider.interceptors.push('OvhNgRequestTaggerInterceptor');
   })
+  .config(($httpProvider) => {
```

Comment:
Explicit annotation could be added:

```suggestion
  .config(/* @ngInject */($httpProvider) => {
```